widget/gridwrap.go

```diff
@@ -54,14 +54,20 @@ type GridWrap struct {
 	// in the GridWrap has been unselected.
 	OnUnselected func(id GridWrapItemID) `json:"-"`
 
-	currentFocus  ListItemID
-	focused       bool
-	scroller      *widget.Scroll
-	selected      []GridWrapItemID
-	itemMin       fyne.Size
-	offsetY       float32
-	offsetUpdated func(fyne.Position)
-	colCountCache int
+	// OnHighlighted is a callback to be notified when a given item
+	// in the GridWrap has been highlighted by keyboard navigation.
```

Comment:
If I followed the conversation, then this is also trigger by mouse hovering, right?

Reply:
Updated the comment to include mouse hover